examples/ublox_receiver_plugin/ublox_receiver_plugin.cpp

```diff
@@ -0,0 +1,58 @@
+#include "ublox_receiver_plugin.h"
+
+#include <boost/thread.hpp>
+
+#include "command.h"
+#include "createReceiverCommand.h"
+#include "receiver_enums.h"
+#include "startCommand.h"
+#include "ublox.h"
+#include "ublox_receiver_view.h"
+
+using namespace ublox;
+
+void UbloxReceiverPlugin::setConfiguration(const QString& version, const QJsonObject& configuration)
+{
+  emit configurationChanged();
+}
+
+QJsonObject UbloxReceiverPlugin::getConfiguration() const
+{
+  return {};
+}
+
+QWidget* UbloxReceiverPlugin::createUI()
+{
+  this->view = new UbloxReceiverView;
+
+  connect(view, &UbloxReceiverView::connectReceiver, [this](int baudRate) {
+    m_skydelNotifier->notify("Connecting to the Ublox receiver");
+    // TODO: port is currently hardcoded... need to put code into CreateUbloxReceiver.cpp to find a port
+    // with a ublox receiver. Also need to change the ui (should ask for the baud rate but not the port)
+    CreateUbloxReceiverCommand command(baudRate, "/dev/ttyACM0");
+    ubloxReceiver = command.execute();
+    view->setReceiverStatus(ReceiverStatus::INACTIVE);
+  });
+
+  connect(view, &UbloxReceiverView::disconnectReceiver, [this]() {
+    m_skydelNotifier->notify("Disconnecting from the Ublox receiver");
+    this->ubloxReceiver->Disconnect();
```

Comment:
No need for `this->`, you can access directly `ubloxReceiver`(like you did with view on the next line).